io.github.tareksander.story_reader.yaml

```diff
@@ -0,0 +1,40 @@
+id: io.github.tareksander.story_reader
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+command: story_reader
+sdk: org.freedesktop.Sdk
+finish-args: [
+  "--share=network",
+  "--socket=pulseaudio",
+  "--socket=fallback-x11",
+  "--socket=wayland",
+  "--share=ipc",
+  "--device=dri"
+]
+cleanup: [
+  "/lib"
```

Comment:
I don't see anything installed in `lib`.

Reply:
I saw something in `/lib` after the flatpak builder finished, I think it extracted debug symbols from the binaries or something? God if it's not needed.

Reply:
one doesn't delete the debug symbols.